File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ApiKeyResponseAllOf.scala

package org.openapitools.client.model


case class ApiKeyResponseAllOf (
    _result: Option[AnyType],
    _msg: Option[AnyType],
    /* The API key that can be used to authenticate as the requested user.  */
    _apiKey: String,
    /* The email address of the user who owns the API key  */
    _email: String
)
object ApiKeyResponseAllOf {
    def toStringBody(var_result: Object, var_msg: Object, var_apiKey: Object, var_email: Object) =
        s"""
        | {
        | "result":$var_result,"msg":$var_msg,"apiKey":$var_apiKey,"email":$var_email
        | }
        """.stripMargin
}

File: javascript-closure-angular/API/Client/ApiKeyResponseAllOf.js
goog.provide('API.Client.ApiKeyResponse_allOf');

/**
 * @record
 */
API.Client.ApiKeyResponseAllOf = function() {}

/**
 * @type {!API.Client.AnyType}
 * @export
 */
API.Client.ApiKeyResponseAllOf.prototype.result;

/**
 * @type {!API.Client.AnyType}
 * @export
 */
API.Client.ApiKeyResponseAllOf.prototype.msg;

/**
 * The API key that can be used to authenticate as the requested user. 
 * @type {!string}
 * @export
 */
API.Client.ApiKeyResponseAllOf.prototype.apiKey;

/**
 * The email address of the user who owns the API key 
 * @type {!string}
 * @export
 */
API.Client.ApiKeyResponseAllOf.prototype.email;

